### ISSUE TITLE
Range sliders don't work on mobile Safari

### ISSUE THREAD

**Issue by open-dynaMIX** [open-dynaMIX/simple-mpv-webui#14]
The goal is to make the position and volume sliders behave identical in safari and android.

This means:

 - Visibly follow when dragging
 - Jump on touch

**Comment by oozorataiyou** [comment]
```css
::-webkit-slider-thumb {
  -webkit-appearance: none;
  width: 1; /* 1 */
  height: 85px;
  background: #999;
  box-shadow: -100vw 0 0 100vw var(--main-fg-color);
  border: 0 solid #999; /* 1 */
}
```

This worked for me 
![IMG_95DB75A9C441-1](https://user-images.githubusercontent.com/3457550/92328306-80803a00-f092-11ea-8f85-3ad76c02e353.jpeg)
![IMG_6378](https://user-images.githubusercontent.com/3457550/92328357-e371d100-f092-11ea-9a27-c88a8e1417f5.JPG)

**Comment by open-dynaMIX** [comment]
Fixed by 21689336be024c857ee1e0f9fa91911e56ab5ffb
